projects/plugins/protect/src/class-rest-controller.php

```diff
@@ -380,10 +382,15 @@ public static function api_get_waf() {
 
 		return new WP_REST_Response(
 			array(
-				'is_seen'    => Jetpack_Protect::get_waf_seen_status(),
-				'is_enabled' => Waf_Runner::is_enabled(),
-				'config'     => Waf_Runner::get_config(),
-				'stats'      => Jetpack_Protect::get_waf_stats(),
+				'wafSupported'        => Waf_Runner::is_supported_environment(),
+				'currentIp'           => IP_Utils::get_ip(),
+				'isSeen'              => Jetpack_Protect::get_waf_seen_status(),
+				'upgradeIsSeen'       => Jetpack_Protect::get_waf_upgrade_seen_status(),
+				'displayUpgradeBadge' => Jetpack_Protect::get_waf_upgrade_badge_display_status(),
+				'isEnabled'           => Waf_Runner::is_enabled(),
+				'config'              => Waf_Runner::get_config(),
+				'stats'               => Jetpack_Protect::get_waf_stats(),
+				'globalStats'         => Waf_Stats::get_global_stats(),
```

Comment:
Ensuring the "waf" API endpoint returns the same data as the initial state "waf" property.